apps/crn-server/serverless.ts

```diff
@@ -871,10 +871,10 @@ const serverlessConfig: AWS = {
             pattern: {
               source: [eventBusSourceContentful],
               'detail-type': [
-                'InterestGroupsPublished',
-                'InterestGroupsUpdated',
-                'InterestGroupsUnpublished',
-                'InterestGroupsDeleted',
+                'GroupsPublished',
+                'GroupsUpdated',
+                'GroupsUnpublished',
+                'GroupsDeleted',
```

Comment:
This isn't right 😅 in Contentful the content type is called InterestGroups so here it should remain as it was

Reply:
Whoops, I changed the wrong file